docs/kit/logistics/README.md

```diff
@@ -0,0 +1,19 @@
+---
+original:
+  authors: Richard Barlow
+  url: https://srobo-legacy.gitbooks.io/student-robotics-kit-logistics
+---
+# Kit Logistics Documentation
+
+This is the documentation of the Student Robotics Kit Logistics Coordinator.
+
+The responsibilities of the Kit Logistics Coordinator are as follows:
+
+* To store the kit
+* To manage the transportation/shipping of the kit
+* To manage the organisation's third part post handling service
+* To maintain a record of the location and state of all Student Robotics assets
```

Comment:
All assets is definitely out of scope of the Kit Logistics Coordinator. 

```suggestion
* To maintain a record of the location and state of all kit
```

Reply:
I've mixed feelings about this. On the one hand I agree that it seems odd that the responsibility of tracking all SR's assets falls to this rather kit-specific role. On the other hand, we _do_ want to have a single system for tracking all things -- having divergent systems would create substantial pain if things moved between them and in terms of training on the systems.

However, I think the real issue here is that this is a merge of legacy docs which have the concept of this as a role at all. In that regard we really want to rework this whole page to be about what documentation you can find here, rather than about the role. Since that's not something we've done while doing the other merges, I'm most tempted to follow that pattern and instead follow-up with a separate PR (or set of PRs) which do that for all the docs.

Reply:
Let's leave this as historically correct, and defer to future updates / kit team to rectify correctly.